custom_components/mikrotik_router/update_types.py

```diff
@@ -34,7 +34,7 @@ class MikrotikUpdateEntityDescription(UpdateEntityDescription):
     ),
     "system_rbfwupdate": MikrotikUpdateEntityDescription(
         key="system_rbfwupdate",
-        name="RouterBoard firmware update",
+        name="RouterBoard Firmware Update",
```

Comment:
this is against HA entity naming policy

Reply:
Sure? All the other update entity names are capitalized. But it doesn´t matter at all because a friendly name can be changed by the user

Reply:
Yes, I discussed this with Frenck last month:
https://developers.home-assistant.io/blog/2022/07/10/entity_naming/
Everything will be renamed to use new standard

Reply:
Great. Reverting this, so :-)